R/derive_param_clinbenefit.R

```diff
@@ -3,12 +3,22 @@
 #' Adds a parameter for clinical benefit/disease control
 #'
 #' @details
-#' Clinical benefit/disease control is first identified for looking for subjects
+#' Clinical benefit/disease control is first identified by looking for subjects
 #' having response status, and then derived for subjects that have at least one
 #' evaluable non-PD response assessment prior to first PD (Progressive Disease)
 #' (i.e., responses exclusive of `NA`, `NE`, `ND`, and `PD`) and after a specified
 #' amount of time from a reference date (`ref_start_window`).
 #'
+#' Note: the user can exclude any additional response values using the `filter_source`
```

Comment:
@mattmarino this has not been updated as per the discussions at https://github.com/pharmaverse/admiralonco/issues/85 - the expectation here was instead of choosing which values could be **excluded**, we'd create a new function argument to enable user to choose with response values may be **included** as eligible for "clinical benefit" with defaults set as `CR`, `PR`, `SD `and `NON-CR/NON-PD`. Then this could replace the `AVALC %in% c("NA", "NE", "ND", "PD")` step of the code and mention of this in the above paragraph.

@bundfussr do you agree? 

p.s. if this update takes longer, as it will need extra unit tests added too, then fine if we take off the list for this release so you have more time.

Reply:
@rossfarrugia , yes, I completely agree. (I just wanted to write the same but you were faster.)

Reply:
Did quite a bit of updating and some unit testing, letting the auto checks run and will check it out in the morning.

Reply:
thanks @mattmarino - feel free to re-request review when you're happy with it